sklearn_numba_dpex/common/_utils.py

```diff
@@ -17,3 +17,7 @@ def _minus(x, y):
 
 def _plus(x, y):
     return x + y
+
+
+def _divide(x, y):
+    return x / y
```

Comment:
What is our convention for division by zero?

Should we use this?
```suggestion
    return 0. if y == 0. else x / y
```

Reply:
I thought it's ok to use whatever numba_dpex decides. 

What is the usual convention ? why return 0. for x / 0 rather than +/-inf ?

Reply:
Returning 0 allow not overshooting other results when addition is used (which is not the case when using ±∞).

Either way, I am mostly happy if the behavior is clear and it not fatal (which might already be the case: I haven't checked).

Anyway, I do not want to block this PR from advancing and let you decide the best decision to take for this case.

Reply:
Anyway currently there aren't usecases with possible division by zero.

Let's decide this when it's an issue with future work and we'll align conventions with existing scikit-learn behavior so we can pass scikit-learn unit tests.